src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -328,6 +336,7 @@ def app_deploy(cmd, client, resource_group, service, name,
                        memory,
                        instance_count,
                        env,
+                       net_core_main_entry_path,
                        target_module,
                        no_wait,
                        file_type,
```

Comment:
We should exclude `NetCoreZip` from displaying build logs as well: https://github.com/Azure/azure-cli-extensions/pull/1371/files#diff-6f0c4fd16bbeedad43a6ad85cfb907e3R376.